parsec/commands/datasets/get_datasets.py

```diff
@@ -1,6 +1,6 @@
 import click
 from parsec.cli import pass_context, json_loads
-from parsec.decorators import custom_exception, list_output
+from parsec.decorators import custom_exception, dict_output
```

Comment:
hmm, this looks wrong, I wonder what happened here.
it's definitely list[dict]
https://bioblend.readthedocs.io/en/latest/api_docs/galaxy/all.html?highlight=get_datasets#bioblend.galaxy.datasets.DatasetClient.get_datasets